### PR TITLE
added info about the 64 char limit for common name

### LINKED ISSUE #10087: Include information regarding the Let's Encrypt 64 character limitation for CN
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https

### What part(s) of the article would you like to see updated?

The generation of Let's Encrypt certification for a Pages custom domain that has a Common Name longer than 64 characters will fail without a clear message being returned to the user.

This limitation should be noted in the Securing your GitHub Pages site with HTTPS documentation.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Fuzzwah**
https://github.com/github/docs/issues/10087

### Why:

Closes https://github.com/github/docs/issues/10087

### What's being changed:

Added info about the 64 char limit for common name, as per RFC 5280

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@Fuzzwah 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by q464pp** [comment]
請問可以拉取合併嗎？

File: content/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ All {% data variables.product.prodname_pages %} sites, including sites that are
 
 {% data reusables.pages.private_pages_are_public_warning %}
 
+{% tip %}
+
+**Tip:** RFC3280 states that the maximum length of the common name should be 64 characters. With this in mind, the entire domain name of your Pages site must be less than 64 characters long for a certificate to be successfully created.
+
+{% endtip %}
```
Comment (skedwards88) [inline content/pages/getting-started-with-github-pages/securing-your-github-pages-site-with-https.md:29]:
```suggestion
{% note %}

**Note:** RFC3280 states that the maximum length of the common name should be 64 characters. Therefore, the entire domain name of your {% data variables.product.prodname_pages %} site must be less than 64 characters long for a certificate to be successfully created.

{% endnote %}
```

**Review by skedwards88 — APPROVED** [review]
Thanks for opening this PR! I made some minor changes. I'll get this merged down for you.

**Pull request merged**